mail_activity_cancel_tracking/static/tests/tours/mail_activity_cancel_tracking.esm.js

```diff
@@ -0,0 +1,31 @@
+/* @odoo-module */
+
+import {registry} from "@web/core/registry";
+
+registry.category("web_tour.tours").add("mail_activity_cancel_tracking_done", {
+    test: true,
+    steps: () => [
+        {
+            trigger: ".o-mail-Activity span:contains('Mark Done')",
+        },
+        {
+            trigger: ".o-mail-ActivityMarkAsDone button[aria-label='Done']",
+        },
+        {
+            trigger: ".o-mail-Message:contains('done'):contains('Play Mario Kart')",
```

Comment:
I think instead doing this in the tour, at the return, you should check the messages in the chatter, and check that there's no cancel message (apart from the done one).